test/unit/TestEval.hs

```diff
@@ -210,6 +212,19 @@ testEval g =
         , testCase
             "try / div by 0"
             ("try {return (1/0)} {return 3}" `evaluatesTo` VInt 3)
+        , testProperty
+            "number of characters"
+            ( \s ->
+                ("chars " <> quote (T.pack s)) `evaluatesToP` VInt (fromIntegral $ length s)
```

Comment:
Oh, is this why you added escaping to `quote`, because you used it to write a test case?  I think we should use a different function here (you should be able to just say `T.pack (show s)`?), remove the escaping from `quote` and `squote`, and better document what `quote` and `squote` are supposed to be for.

Reply:
Yes, that was what I did at first, but got confused by a different mistake. :sweat_smile: 

Now it should be fixed.